### DISCUSSION TITLE
Equations for physiological parameters and PBPK model

### DISCUSSION THREAD

**Discussion by hsuanpin** [Open-Systems-Pharmacology/Forum#794] [Questions & Problems]
Hi,

I would like to know where I can find the equations for each physiological parameter. For example, equations that describe organ volume as a function of age, weight, gender, etc. Also, where can I find the differential equations for the PBPK model. Thank you.

**Comment by JanSchlender** [comment]
Maybe this discussion is of help to your question:
https://github.com/Open-Systems-Pharmacology/Forum/issues/360#issuecomment-531397108

**Comment by hsuanpin** [comment]
Hi Jan,

Thank you for the information. However, I still have questions about how the software generates physiological parameters. Let's take organ volume as an example. I know that organ volume is from literature reported values for a specific age (i.e., ICRP report brain weight for different age groups). Do you use body weight-based organ volume (i.e., mL/kg) or other methods? If so, how do you get this body weight-based organ volume from the literature (they usually reported total volume but not body-weight based organ volume). In addition, how about user-defined age that is not reported in the literature (i.e., 5.6 yr)? Do you do interpolation to get the physiological parameter or other methods, and how? Thank you.

Best regards,
Ping

**Comment by JanSchlender** [comment]
Hi Ping,

What do you mean by body weight-based organ volumes? The Create Individual algorithm is discribed here:
https://github.com/Open-Systems-Pharmacology/OSPSuite.Documentation/wiki/Create-Individual-Algorithm 
and in a publication by Willmann et al
https://pubmed.ncbi.nlm.nih.gov/17431751/
These references also explain how interpolation between bin is conducted.

Best,

Jan

**Comment by StephanSchaller** [comment]
It may be good to add the details of this section in the documentation (i.e. the online manual: https://docs.open-systems-pharmacology.org/).

**Comment by hsuanpin** [comment]
> Hi Ping,
> 
> What do you mean by body weight-based organ volumes? The Create Individual algorithm is discribed here:
> https://github.com/Open-Systems-Pharmacology/OSPSuite.Documentation/wiki/Create-Individual-Algorithm
> and in a publication by Willmann et al
> https://pubmed.ncbi.nlm.nih.gov/17431751/
> These references also explain how interpolation between bin is conducted.
> 
> Best,
> 
> Jan

**Comment by hsuanpin** [comment]
Hi Jan,

Thank you for the reference. The body weight-based organ volume is how much organ volume per kg of body weight (i.e., liver volume for fault is 32 mL/kg, calculated by 2358 mL/73 kg). This would be similar to % of BW in table I in the reference you provided. It seems like the organ volume is derived based on height, body weight, and BMI. However, I have questions about how the software obtain physiological parameter for pediatrics. Do you use **ADULT** mean organ weight reported by ICRP, and derive target organ weight (i.e., calculated by equation 1 in the paper) using **PEDIATRICS**' height, body weight, and BMI? However, ICRP also provided physiological parameters for pediatrics, so do the software use this information?

Thank you again for your help!

Best regards,
Ping
